src/lib/viewers/controls/annotations/annotations-button-region-tooltips/PersistentOnboardingBoxEditAnnotationsTooltip.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { useContext, PropsWithChildren } from 'react';
+import classNames from 'classnames';
+
+// TODO: Compare bundle sizes and possibly re-write here
```

Comment:
I agree, we should do this before proceeding. It looks like this component has many other dependencies.

Reply:
I'll do that, when other issues mentioned here are resolved. I added that to TODO section in PR's description.